src/scenes/AcceleratorRouter/Scoring/ScoreEntry.tsx

```diff
@@ -0,0 +1,148 @@
+import { useState } from 'react';
+
+import { Box, Button, Typography, useTheme } from '@mui/material';
+
+import TextArea from 'src/components/common/TextArea';
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { Score, ScoreValue, ScoreValues, getScoreCategory, getScoreColors, getScoreValue } from 'src/types/Score';
+
+type ScoringEntryProps = {
+  disabled?: boolean;
+  onChange: (value: ScoreValue) => void;
+  onChangeText: (id: string, value: unknown) => void;
+  phase: string;
+  qualitativeInfo: string;
+  score: Score;
+};
+
+const ScoreEntry = ({ disabled, onChange, onChangeText, phase, score, qualitativeInfo }: ScoringEntryProps) => {
+  const theme = useTheme();
+  const scoreCategory = getScoreCategory(score.category);
+  const { activeLocale } = useLocalization();
+
+  return activeLocale ? (
+    <Box sx={{ margin: '16px 0', width: 1 }}>
+      <Box sx={{ marginBottom: '8px' }}>
+        <Typography sx={{ color: theme.palette.TwClrBaseBlack, fontWeight: 500 }}>{phase}</Typography>
+        <Typography sx={{ color: theme.palette.TwClrBaseBlack, fontSize: '18px', fontWeight: 500 }}>
+          {scoreCategory} {strings.SCORE}
+        </Typography>
+      </Box>
+
+      <ScoreControl disabled={disabled} onChange={onChange} score={score} />
+
+      <Box>
+        <Typography sx={{ color: theme.palette.TwClrTxtSecondary, fontSize: '14px', marginBottom: '4px' }}>
+          {strings.QUALITATIVE_INFO}
+        </Typography>
+        {disabled ? (
+          <Typography sx={{ color: theme.palette.TwClrBaseBlack }}>{qualitativeInfo}</Typography>
+        ) : (
+          <TextArea id='qualitative-info' label='' value={qualitativeInfo} onChange={onChangeText} />
+        )}
+      </Box>
+    </Box>
+  ) : null;
+};
+
+export default ScoreEntry;
+
+type ScoreControlProps = {
+  disabled?: boolean;
+  onChange: (value: ScoreValue) => void;
+  score: Score;
+};
+
+const ScoreControl = ({ disabled, onChange, score }: ScoreControlProps) => {
+  const theme = useTheme();
+  const castScoreValue = Number(score.value) as ScoreValue;
+  const [scoreValue, setScoreValue] = useState<ScoreValue>(castScoreValue);
+  const scoreValueLabel = getScoreValue(scoreValue);
+  const scoreColors = getScoreColors(scoreValue, theme);
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: theme.palette.TwClrBaseGray050,
+        borderRadius: '8px',
+        display: 'inline-flex',
+        marginBottom: theme.spacing(2),
+        padding: '12px',
+      }}
```

Comment:
There's also a lot of usage of the `sx` prop to set styles, I can move many of these to style props on the component if that's preferred.

Reply:
Ok that's kind of what I was thinking, but wasn't sure if we had a preference. Thanks!